torch/csrc/jit/codegen/cuda/root_domain_map.h

```diff
@@ -206,7 +206,7 @@ class TORCH_CUDA_API ComputeAtRootDomainMap : public RootDomainMap {
  public:
   //! Builds a mapping table by analyzing the current
   //! fusion. Overwrite a previous table if any.
-  void build();
+  void build(bool map_through_reduction = false);
```

Comment:
Can you add a comment on the new parameter?

Reply:
Added Thanks!